tests/scaletest/scale_test.go

```diff
@@ -769,6 +777,113 @@ func UpdateIngressParallelWithNodeReboot(t *testing.T) {
 	REBOOTNODE = false
 }
 
+func CreateServiceTypeLBWithApp(t *testing.T, numPods int, numOfServices int, appNameLB string, serviceNamePrefixLB string, aviObjPrefix string) []string {
+	g := gomega.NewGomegaWithT(t)
+	t.Logf("Creating a %v deployment with %v replicas", appNameLB, numPods)
+	err := lib.CreateApp(appNameLB, namespace, numPods)
+	if err != nil {
+		t.Fatalf("ERROR : Could not create deployment for service type LB support as %v", err)
+	}
+
+	t.Logf("Creating %v services of type LB", numOfServices)
+	servicesCreated, port, err := lib.CreateLBService(serviceNamePrefixLB, appNameLB, namespace, numOfServices)
+	if err != nil {
+		t.Fatalf("ERROR : %v", err)
```

Comment:
We can make it more verbose, e.g. above in case of failure for deployment creation.

Reply:
Will display the message clearly